Swiftfin/Components/AlphaPicker/AlphaPickerButton.swift

```diff
@@ -0,0 +1,76 @@
+//
+// Swiftfin is subject to the terms of the Mozilla Public
+// License, v2.0. If a copy of the MPL was not distributed with this
+// file, you can obtain one at https://mozilla.org/MPL/2.0/.
+//
+// Copyright (c) 2023 Jellyfin & Jellyfin Contributors
+//
+
+import Defaults
+import SwiftUI
+
+extension AlphaPickerView {
+
+    struct AlphaPickerButton: View {
+
+        @Default(.accentColor)
+        private var accentColor
+
+        private let nameStartsWith: String
+        private let activated: Bool
+        private let viewModel: FilterViewModel
+        private var onSelect: () -> Void
+
+        var body: some View { 
+            Button {
+                action: do {
```

Comment:
I believe you may be confused on trailing syntax, this is a [labeled statement](https://docs.swift.org/swift-book/documentation/the-swift-programming-language/statements/#Labeled-Statement) to help with control flow. While the syntax itself looks similar to a [trailing closure label](https://www.fuckingmultipletrailingclosuresyntax.com/), they are completely different.

Remove the `action: do` block.

Reply:
Gotcha! This should be updated now and I've test it to confirm this does not change the functionality.